backend/app/views/spree/admin/customer_returns/new.html.erb

```diff
@@ -33,7 +33,7 @@
         </fieldset>
 
         <%= f.field_container :stock_location do %>
-          <%= f.label :stock_location, Spree.t(:stock_location) %>
+          <%= f.label :stock_location_id %>
```

Comment:
To be more clear about this:

```erb
<%= f.label :stock_location, Spree::StockLocation.model_name.human %>
```

If this also fixes the select2 label problem, great